file-share-service-desktop-client/Modules/Admin/AdminView.xaml

```diff
@@ -16,13 +16,13 @@
                 <Setter Property="FontSize" Value="16" />
             </Style>
 
-            <Style TargetType="TextBlock" x:Key="KeyValuePairValueTextBlockStyle">
+            <Style TargetType="{x:Type TextBox}" x:Key="KeyValuePairValueTextBlockStyle">
                 <Setter Property="FontSize" Value="16" />
                 <Setter Property="Margin" Value="4 0" />
-                <Setter Property="TextTrimming" Value="CharacterEllipsis" />
+                <!--<Setter Property="TextTrimming" Value="CharacterEllipsis" />-->
                 <Setter Property="TextWrapping" Value="NoWrap" />
                 <Setter Property="FontWeight" Value="Bold" />
-                <Setter Property="ToolTip" Value="{Binding RelativeSource={RelativeSource Self}, Path=Text}" />
+                <!--<Setter Property="ToolTip" Value="{Binding RelativeSource={RelativeSource Self}, Path=Text}" />-->
```

Comment:
should probably just be deleted

Reply:
Commented code removed

Reply:
Where ever possible I have removed the inline style